AnnoyancesFilter/sections/popups.txt

```diff
@@ -110,6 +110,8 @@ diabetdieta.ru##.subscribe-from-block-msg
 ehow.com#$#.component-modal-newsletter { display: none!important; }
 ehow.com#$#body { overflow: auto !important; }
 yahoo.co.jp###Perorin
+cyprusjobcentre.com###OverMaskNA
+cyprusjobcentre.com###LoaderDiv
```

Comment:
That popup is created by jQuery.NagAds.min.js Better block it.

Reply:
I suppose there is no need for these rules if you block the script.